api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -70,9 +69,52 @@ message ListenersConfigDump {
     api.v2.Listener listener = 2;
 
     // The timestamp when the Listener was last updated.
+    // FIXME: reviewer we may eventually want last updated and last _successfully_ updated.
+    // Given similarity between last_updated (attempt or success) and
+    // last_update should this maybe be last_update_attempt so last_update
+    // matches?
     google.protobuf.Timestamp last_updated = 3;
   }
 
+  // Describes a dynamically loaded listener via the LDS API.
+  // FIXME rename? because otherwise v2->alpha is super confusing but this seems
+  // like the right name :-/
+  message DynamicListener {
+    // FIXME worth splitting name out for easy reference, duplicate cruft?
```

Comment:
Yeah, this is probably reasonable, in particular as this is going to be the basis of JSON dump, so I'm +1 on what you have here.

Reply:
The other option is to just use a map below which is name -> DynamicListener?